backend/app/interior/infra/db_models/interior.py

```
@@ -0,0 +1,40 @@
+from datetime import datetime
+from typing import Optional, Dict, Any, List
```

Ruff (0.12.2)

`typing.Dict` imported but unused

Remove unused import

(F401)